tests/fixtures/migration/feature_list

```diff
@@ -478,6 +440,60 @@
         ]
       }
     ],
+    "relationships_info": [
+      {
+        "_id": {
+          "$oid": "6470894ddc5e0118a048d546"
+        },
+        "relationship_type": "child_parent",
+        "entity_id": {
+          "$oid": "64708932ea4c4876a77d2b85"
+        },
+        "related_entity_id": {
+          "$oid": "64708937ea4c4876a77d2b89"
+        },
+        "relation_table_id": {
+          "$oid": "64708925ea4c4876a77d2b83"
+        }
+      },
+      {
+        "_id": {
+          "$oid": "6470894ddc5e0118a048d540"
+        },
+        "relationship_type": "child_parent",
+        "entity_id": {
+          "$oid": "64708932ea4c4876a77d2b85"
+        },
+        "related_entity_id": {
+          "$oid": "64708936ea4c4876a77d2b88"
+        },
+        "relation_table_id": {
+          "$oid": "64708925ea4c4876a77d2b83"
+        }
+      },
+      {
+        "_id": {
+          "$oid": "6470893ddc5e0118a048d53a"
+        },
+        "relationship_type": "child_parent",
+        "entity_id": {
+          "$oid": "64708934ea4c4876a77d2b86"
+        },
+        "related_entity_id": {
+          "$oid": "64708932ea4c4876a77d2b85"
+        },
+        "relation_table_id": {
+          "$oid": "6470891bea4c4876a77d2b81"
+        }
+      }
+    ],
+    "readiness_distribution": [
+      {
+        "readiness": "PRODUCTION_READY",
+        "count": 1
+      }
+    ],
+    "deployed": false,
```

Comment:
Update this fixture to use older record (without `supported_serving_entity_ids` attribute).